ludwig/benchmarking/summary_dataclasses.py

```diff
@@ -43,15 +47,31 @@ class MetricDiff:
     # Value of the metric in the experimental experiment.
     experimental_value: float
 
-    # Experimental_value - base_value.
+    # experimental_value - base_value.
     diff: float
 
     # Percentage of change the metric with respect to base_value.
-    diff_percentage: float
+    diff_percentage: Union[float, str]
+
+    def __post_init__(self):
+        """Add human-readable string representations to the field."""
+
+        if "memory" in self.name:
+            self.base_value_str = _format_memory(self.base_value)
```

Comment:
These methods from torch have leading underscores, which suggests that they should be interpreted and treated as private/internal methods.

If we want to use the functionality verbatim, then we should copy the implementation and attribute instead of using the internal method directly, or look to see if they have a public API.